test/specs/Loader.spec.js

```diff
@@ -33,8 +33,10 @@ beforeEach(function() {
       ],
       defaultTopicId: 'sometopic',
       translationFallbackCode: 'somelang',
-      defaultExtent: [420000, 30000,900000, 350000],
```

Comment:
It's a mistake, I had `defaultExtent` twice in my branch. I will cancel the deletion.